Started GET "/" for 127.0.0.1 at 2020-08-11 15:54:19 -0300
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by HomeController#index as HTML
  Rendering home/index.html.erb within layouts/application
  Rendered home/index.html.erb within layouts/application (Duration: 38.1ms | Allocations: 26677)
[Webpacker] Compiling...
[Webpacker] Compilation failed:
ERROR: [Errno 21] Is a directory: 'bin'
CRITICAL:root:Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cliapp/app.py", line 182, in _run
    self.settings.load_configs()
  File "/usr/lib/python3/dist-packages/cliapp/settings.py", line 829, in load_configs
    self._read_ini(pathname, f)
  File "/usr/lib/python3/dist-packages/cliapp/settings.py", line 838, in _read_ini
    cp.readfp(f)
  File "/usr/lib/python3.8/configparser.py", line 763, in readfp
    self.read_file(fp, source=filename)
  File "/usr/lib/python3.8/configparser.py", line 718, in read_file
    self._read(f, source)
  File "/usr/lib/python3.8/configparser.py", line 1082, in _read
    raise MissingSectionHeaderError(fpname, lineno, line)
configparser.MissingSectionHeaderError: File contains no section headers.
file: '/home/diana/projects/plataforma-headhunters-treina-dev/config/webpack/development.js', line: 1
"process.env.NODE_ENV = process.env.NODE_ENV || 'development'\n"

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cliapp/app.py", line 182, in _run
    self.settings.load_configs()
  File "/usr/lib/python3/dist-packages/cliapp/settings.py", line 829, in load_configs
    self._read_ini(pathname, f)
  File "/usr/lib/python3/dist-packages/cliapp/settings.py", line 838, in _read_ini
    cp.readfp(f)
  File "/usr/lib/python3.8/configparser.py", line 763, in readfp
    self.read_file(fp, source=filename)
  File "/usr/lib/python3.8/configparser.py", line 718, in read_file
    self._read(f, source)
  File "/usr/lib/python3.8/configparser.py", line 1082, in _read
    raise MissingSectionHeaderError(fpname, lineno, line)
configparser.MissingSectionHeaderError: File contains no section headers.
file: '/home/diana/projects/plataforma-headhunters-treina-dev/config/webpack/development.js', line: 1
"process.env.NODE_ENV = process.env.NODE_ENV || 'development'\n"

Completed 200 OK in 636ms (Views: 629.4ms | ActiveRecord: 0.0ms | Allocations: 59196)


Started GET "/" for 127.0.0.1 at 2020-08-11 15:54:22 -0300
Processing by HomeController#index as HTML
  Rendering home/index.html.erb within layouts/application
  Rendered home/index.html.erb within layouts/application (Duration: 2.5ms | Allocations: 1056)
[Webpacker] Everything's up-to-date. Nothing to do
Completed 200 OK in 17ms (Views: 15.4ms | ActiveRecord: 0.0ms | Allocations: 12948)


Started GET "/users/sign_in" for 127.0.0.1 at 2020-08-11 15:54:25 -0300
Processing by Users::SessionsController#new as HTML
  Rendering devise/sessions/new.html.erb within layouts/application
  Rendered devise/shared/_links.html.erb (Duration: 0.9ms | Allocations: 591)
  Rendered devise/sessions/new.html.erb within layouts/application (Duration: 21.2ms | Allocations: 11206)
[Webpacker] Everything's up-to-date. Nothing to do
Completed 200 OK in 55ms (Views: 34.7ms | ActiveRecord: 4.9ms | Allocations: 30260)


Started GET "/users/sign_up" for 127.0.0.1 at 2020-08-11 15:54:28 -0300
Processing by Users::RegistrationsController#new as HTML
  Rendering devise/registrations/new.html.erb within layouts/application
  Rendered devise/shared/_error_messages.html.erb (Duration: 0.6ms | Allocations: 198)
  Rendered devise/shared/_links.html.erb (Duration: 0.4ms | Allocations: 111)
  Rendered devise/registrations/new.html.erb within layouts/application (Duration: 4.6ms | Allocations: 1698)
[Webpacker] Everything's up-to-date. Nothing to do
Completed 200 OK in 22ms (Views: 20.5ms | ActiveRecord: 0.0ms | Allocations: 15224)


Started POST "/users" for 127.0.0.1 at 2020-08-11 15:54:51 -0300
Processing by Users::RegistrationsController#create as HTML
  Parameters: {"authenticity_token"=>"PF1u563b5gFzHivt6UjBSahbLT7FZXOgEae80H1HYWfrFokwYsFu1c60atPDxAhXQpTEqjp6UeCG5V5bex1cPQ==", "user"=>{"email"=>"dianaregina22@gmail.com.br", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]"}}
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  [1m[36mUser Exists? (0.4ms)[0m  [1m[34mSELECT 1 AS one FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "dianaregina22@gmail.com.br"], ["LIMIT", 1]]
  [1m[35m (0.2ms)[0m  [1m[31mROLLBACK[0m
  Rendering devise/registrations/new.html.erb within layouts/application
  Rendered devise/shared/_error_messages.html.erb (Duration: 0.3ms | Allocations: 200)
  Rendered devise/shared/_links.html.erb (Duration: 0.1ms | Allocations: 79)
  Rendered devise/registrations/new.html.erb within layouts/application (Duration: 1.6ms | Allocations: 1146)
[Webpacker] Everything's up-to-date. Nothing to do
Completed 200 OK in 128ms (Views: 10.2ms | ActiveRecord: 0.8ms | Allocations: 22130)


Started POST "/users" for 127.0.0.1 at 2020-08-11 15:55:02 -0300
Processing by Users::RegistrationsController#create as HTML
  Parameters: {"authenticity_token"=>"vA7pF0O0YDOlZH26hb+J3HHGoEVNigLx7uRcWty9v90gUn6+pfGbFEw/mstnzYljRrXtk66Q15HT6mSirKeKjQ==", "user"=>{"email"=>"dianaregina22@gmail.com.br", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]"}}
  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
  [1m[36mUser Exists? (0.3ms)[0m  [1m[34mSELECT 1 AS one FROM "users" WHERE "users"."email" = $1 LIMIT $2[0m  [["email", "dianaregina22@gmail.com.br"], ["LIMIT", 1]]
  [1m[36mUser Create (0.4ms)[0m  [1m[32mINSERT INTO "users" ("email", "encrypted_password", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"[0m  [["email", "dianaregina22@gmail.com.br"], ["encrypted_password", "$2a$11$eyOXRm6h2xrCQuM9dJygV.e869D7vMwH9P3wZCMYDpTwWUCbpmrAa"], ["created_at", "2020-08-11 18:55:02.694388"], ["updated_at", "2020-08-11 18:55:02.694388"]]
  [1m[35m (13.3ms)[0m  [1m[35mCOMMIT[0m
Redirected to http://localhost:3000/
Completed 302 Found in 138ms (ActiveRecord: 14.2ms | Allocations: 7352)


Started GET "/" for 127.0.0.1 at 2020-08-11 15:55:02 -0300
Processing by HomeController#index as HTML
  [1m[36mUser Load (0.3ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 ORDER BY "users"."id" ASC LIMIT $2[0m  [["id", 3], ["LIMIT", 1]]
  â†³ app/controllers/home_controller.rb:5:in `index'
Redirected to http://localhost:3000/candidates/new
Completed 302 Found in 4ms (ActiveRecord: 0.6ms | Allocations: 1434)


Started GET "/candidates/new" for 127.0.0.1 at 2020-08-11 15:55:02 -0300
Processing by CandidatesController#new as HTML
  [1m[36mUser Load (0.2ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 ORDER BY "users"."id" ASC LIMIT $2[0m  [["id", 3], ["LIMIT", 1]]
  â†³ app/controllers/candidates_controller.rb:124:in `authenticate_visitor'
An enum element in Candidate uses the prefix 'not_'. This will cause a conflict with auto generated negative scopes.
  [1m[36mCandidate Load (0.3ms)[0m  [1m[34mSELECT "candidates".* FROM "candidates" WHERE "candidates"."user_id" = $1 LIMIT $2[0m  [["user_id", 3], ["LIMIT", 1]]
  â†³ app/controllers/candidates_controller.rb:110:in `find_candidate'
  Rendering candidates/new.html.erb within layouts/application
  [1m[36mLevel Load (0.3ms)[0m  [1m[34mSELECT "levels".* FROM "levels"[0m
  â†³ app/views/candidates/_form.html.erb:39
  Rendered candidates/_form.html.erb (Duration: 12.1ms | Allocations: 6501)
  Rendered candidates/new.html.erb within layouts/application (Duration: 12.8ms | Allocations: 6800)
[Webpacker] Everything's up-to-date. Nothing to do
  [1m[36mCACHE Candidate Load (0.0ms)[0m  [1m[34mSELECT "candidates".* FROM "candidates" WHERE "candidates"."user_id" = $1 LIMIT $2[0m  [["user_id", 3], ["LIMIT", 1]]
  â†³ app/views/layouts/application.html.erb:28
Completed 200 OK in 48ms (Views: 22.3ms | ActiveRecord: 4.9ms | Allocations: 32602)


Started DELETE "/users/sign_out" for 127.0.0.1 at 2020-08-11 15:55:08 -0300
Processing by Users::SessionsController#destroy as HTML
  Parameters: {"authenticity_token"=>"i2BwL7PN4dOOnNkPZp9PJgLAvHHSc0HwdN+JAB/+q64XPOeGVYga9GfHPn6E7U+ZNbPxpzFplJBJ0bH4b+Se/g=="}
  [1m[36mUser Load (0.4ms)[0m  [1m[34mSELECT "users".* FROM "users" WHERE "users"."id" = $1 ORDER BY "users"."id" ASC LIMIT $2[0m  [["id", 3], ["LIMIT", 1]]
Redirected to http://localhost:3000/
Completed 302 Found in 13ms (ActiveRecord: 0.4ms | Allocations: 2706)


Started GET "/" for 127.0.0.1 at 2020-08-11 15:55:08 -0300
Processing by HomeController#index as HTML
  Rendering home/index.html.erb within layouts/application
  Rendered home/index.html.erb within layouts/application (Duration: 1.4ms | Allocations: 1062)
[Webpacker] Everything's up-to-date. Nothing to do
Completed 200 OK in 15ms (Views: 14.4ms | ActiveRecord: 0.0ms | Allocations: 12973)


